src/main.ts

```diff
@@ -4,18 +4,26 @@ import OpenAI from "openai";
 import { Octokit } from "@octokit/rest";
 import parseDiff, { Chunk, File } from "parse-diff";
 import minimatch from "minimatch";
+import { addSyntheticLeadingComment } from "typescript";
```

Comment:
The import statement for `addSyntheticLeadingComment` from `typescript` seems unnecessary as it is not used in the provided code. Consider removing it to keep the code clean.